cargo-dist/tests/gallery/dist.rs

```diff
@@ -209,21 +209,51 @@ impl<'a> TestContext<'a, Tools> {
         // read/analyze installers
         eprintln!("loading results...");
         let app_name = &self.repo.app_name;
-        let ps_installer = Utf8PathBuf::from(format!("target/distrib/{app_name}-installer.ps1"));
-        let sh_installer = Utf8PathBuf::from(format!("target/distrib/{app_name}-installer.sh"));
-        let rb_installer = Utf8PathBuf::from(format!("target/distrib/{app_name}.rb"));
+        let target_dir = Utf8PathBuf::from("target/distrib");
+        let ps_installer = Utf8PathBuf::from(format!("{target_dir}/{app_name}-installer.ps1"));
+        let sh_installer = Utf8PathBuf::from(format!("{target_dir}/{app_name}-installer.sh"));
+        let homebrew_installer = Self::load_file_with_extension(target_dir.clone(), ".rb");
         let npm_installer =
-            Utf8PathBuf::from(format!("target/distrib/{app_name}-npm-package.tar.gz"));
+            Utf8PathBuf::from(format!("{target_dir}/{app_name}-npm-package.tar.gz"));
 
         Ok(DistResult {
             test_name: test_name.to_owned(),
             shell_installer_path: sh_installer.exists().then_some(sh_installer),
             powershell_installer_path: ps_installer.exists().then_some(ps_installer),
-            homebrew_installer_path: rb_installer.exists().then_some(rb_installer),
+            homebrew_installer_path: homebrew_installer,
             npm_installer_package_path: npm_installer.exists().then_some(npm_installer),
         })
     }
 
+    fn load_file_with_extension(dirname: Utf8PathBuf, extension: &str) -> Option<Utf8PathBuf> {
+        let files = Self::load_files_with_extension(dirname, extension);
+        let number_found = files.len();
+        assert!(
+            number_found <= 1,
+            "found {} files with the extension {}, expected 1 or 0",
+            number_found,
+            extension
+        );
+        files.first().cloned()
+    }
+
+    fn load_files_with_extension(dirname: Utf8PathBuf, extension: &str) -> Vec<Utf8PathBuf> {
```

Comment:
I suggest

```suggestion
    fn find_files_with_suffix(dirname: &Utf8Path, suffix: &str) -> Vec<Utf8PathBuf> {
```

(and a bunch of fallout changes from that)